indico/core/settings/proxy.py

```diff
@@ -203,3 +203,19 @@ def __set__(self, obj, value):
 
     def __delete__(self, obj):
         self.proxy.delete(*self._make_args(obj, self.name))
+
+
+class ProxyProperty(object):
```

Comment:
Not sure about the naming:
  * SettingProperty - proxy property that maps to a setting;
  * ProxyProperty - proxy property that maps to an attribute of a proxied attribute;

What about `AttributeProperty`?